src/components/expanded-state/LiquidityPoolExpandedState.js

```diff
@@ -5,7 +5,7 @@ import { getSoftMenuBarHeight } from 'react-native-extra-dimensions-android';
 import { useDispatch } from 'react-redux';
 import { UniBalanceHeightDifference } from '../../hooks/charts/useChartThrottledPoints';
 import deviceUtils from '../../utils/deviceUtils';
-import EdgeFade from '../discover-sheet/EdgeFade';
+import EdgeFade from '../discover/EdgeFade';
```

Comment:
nit: should prob move this to somewhere in components since its being reused

Reply:
I can move it one level down for sure